docs/staging/dmb/dmb-aspects-of-a-good-application.md

```diff
@@ -0,0 +1,120 @@
+(dmb-aspects-of-a-good-application)=
+# Aspects of a good DMB application
+
+```{note}
+[Source page](https://wiki.ubuntu.com/RobieBasak/DMB/Expectations)
+
+```
+
+Since every applicant is unique, what makes a "good" DMB application can be flexible. No two applications are ever exactly the same, and everyone has a different position to some extent.
+
+If you need different considerations for some reason, make sure to explain this in your application and we are happy to consider it.
+
+## Track record
+
+Applicants must have an existing track record of sponsored uploads to the packages they are requesting direct upload access to.
+The DMB normally approves applications either to reduce contributor friction in needing sponsorship or to save the work of sponsors.
+
+Example:
+
+* Alice is working diligently on Openstack packages, getting many sponsored uploads every cycle. After a second cycle, in which only minimal / minor changes needed to happen, Alice feels confident in applying for cloud packageset upload rights.
```

Comment:
"in which only minimal / minor *review* changes needed to happen" is I think what you mean, as opposed to packaging changes in general?

Reply:
yes, "review triggered" is even more clear.
Thanks - added